app/views/records/edit_fields/_creator.html.erb

```diff
@@ -43,7 +44,7 @@
 				{ field_type: :text, field_slug: :creator_family_name, field_args: { data: { required: true, required_group: :creator_name, on_blur: :toggle_required_group } } },
 				{ field_type: :text, field_slug: :creator_given_name, field_args: { data: { required: true, required_group: :creator_name, on_blur: :toggle_required_group } } },
 				{ field_type: :text, field_slug: :creator_orcid },
-				{ field_type: :select, field_slug: :creator_institutional_relationship, select_options: ['Staff member', 'Research associate'], field_args: { multiple: true } }] %>
+				{ field_type: :select, field_slug: :creator_institutional_relationship, select_options: HykuAddons::InstitutionalRelationshipService.new.select_active_options, field_args: { multiple: true } }] %>
```

Comment:
Do you want to pass the model along to the QA service?

Reply:
@cjcolvar thats done.